pkg/collector/corechecks/cluster/orchestrator/collectors/k8s/clusterrole.go

```diff
@@ -100,7 +100,7 @@ func (c *ClusterRoleCollector) Process(rcfg *collectors.CollectorRunConfig, list
 
 	result := &collectors.CollectorRunResult{
 		Result:             processResult,
-		ResourcesListed:    len(c.processor.Handlers().ResourceList(ctx, list)),
+		ResourcesListed:    processed,
```

Comment:
I think it will break this metric https://github.com/DataDog/datadog-agent/blob/main/pkg/collector/corechecks/cluster/orchestrator/collector_bundle.go#L368 as `ResourcesListed` will be always equal to `ResourcesProcessed`

Reply:
since the beginning the two have been equal because we return the number of resource in the list in this common code https://github.com/DataDog/datadog-agent/blob/49ad0d9a4b637eeb5f6da61fa76230ee8cc0175f/pkg/collector/corechecks/cluster/orchestrator/processors/processor.go#L265

Reply:
updated